src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -154,12 +159,116 @@ class BrandedCheckoutStep1Controller {
   checkSuccessfulSubmission () {
     if (every(this.submission, 'completed')) {
       if (every(this.submission, { error: false })) {
-        this.next()
+        if (this.useV3 === 'true') {
+          this.submitOrderInternal()
+        } else {
+          this.next()
+        }
       } else {
         this.submitted = false
       }
     }
   }
+
+  loadCart () {
+    this.errorLoadingCart = false
+
+    // Return the observable instead of subscribing here
+    return this.cartService.get().pipe(
+      tap(data => {
+        // Setting cart data and analytics
+        this.cartData = data
+        this.brandedAnalyticsFactory.saveCoverFees(this.orderService.retrieveCoverFeeDecision())
+        this.brandedAnalyticsFactory.saveItem(this.cartData.items[0])
+        this.brandedAnalyticsFactory.addPaymentInfo()
+        this.brandedAnalyticsFactory.reviewOrder()
+        console.log('done loading cart')
```

Comment:
The four logs in this file and one in `order.service.js` should be removed before merging.